Sources/FoundationEssentials/String/String+Internals.swift

```diff
@@ -28,51 +28,23 @@ extension String {
             throw CocoaError.errorWithFilePath(.fileReadInvalidFileName, "")
         }
 
-        // 1. Normalize the path first.
-
-        var path = self
+        var iter = self.makeIterator()
+        let bLeadingSlash = if iter.next() == "/", iter.next()?.isLetter ?? false, iter.next() == ":" { true } else { false }
```

Comment:
Should we use the UTF-8 view here too? (`self.utf.makeIterator()`, `iter.net() == ._slash`, etc.)

Reply:
Done